e2e/tests/components/dashboard/dashboard.e2e-spec.ts

```diff
@@ -84,6 +116,9 @@ describe('Dashboard Tests', () => {
 
         expect(await page.getWidgetLocationValue(widget4, 'top')).toBe(0);
         expect(await page.getWidgetLocationValue(widget4, 'left')).toBe(831);
+
+        expect(await page.getLayoutOutput()).toBe('[ { "id": "analytics-1-widget", "col": 0, "row": 1, "colSpan": 4, "rowSpan": 2 }, { "id": "subscription-widget", "col": 0, "row": 0, "colSpan": 2, "rowSpan": 1 }, { "id": "users-widget", "col": 2, "row": 0, "colSpan": 1, "rowSpan": 1 }, { "id": "alert-widget", "col": 3, "row": 0, "colSpan": 1, "rowSpan": 1 } ]');

```

Comment:
Would it be better to have this value stored as an object and then use `JSON.stringify(...)`. That way anyone having to modify one of the values in the future wouldn't need to edit a rather large string?